src/FirebaseArduino.cpp

```diff
@@ -61,69 +61,69 @@ void FirebaseArduino::setBool(const String& path, bool value) {
 }
 
 void FirebaseArduino::setString(const String& path, const String& value) {
-  JsonVariant json(value);
+  JsonVariant json(value.c_str());
```

Comment:
This one is odd because it is a pre-existing bug. If you pass JasonVariant an Arduino String it will always interpret it as bool (at least it does for the string in FirebaseDemo).